development/markdown-index/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.19.3-alpine3.16 as builder
```

Comment:
There is newer version of Alpine available, 3.17

Reply:
golang:1.19.3-alpine3.16 is the newest one, anyway dependabot should update it